### PR TITLE
chore(license): Right license in .editorconfig

### PR DESCRIPTION
**Pull request by lemaitre-aneo**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -65,7 +65,7 @@ dotnet_style_qualification_for_field = false:suggestion
...
+file_header_template = This file is part of the ArmoniK project\n\nCopyright (C) ANEO, 2022-$CURRENT_YEAR$. All rights reserved.\n  W. Kirschenmann   <wkirschenmann@aneo.fr>\n  J. Gurhem         <jgurhem@aneo.fr>\n  D. Dubuc          <ddubuc@aneo.fr>\n  L. Ziane Khodja   <lzianekhodja@aneo.fr>\n  F. Lemaitre       <flemaitre@aneo.fr>\n  S. Djebbar        <sdjebbar@aneo.fr>\n  J. Fonseca        <jfonseca@aneo.fr>\n  D. Brasseur       <dbrasseur@aneo.fr>\n\nLicensed under the Apache License, Version 2.0 (the "License")\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an "AS IS" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License.\n
```
Comment (ddubuc) [inline .editorconfig:68]:
```suggestion
file_header_template = This file is part of the ArmoniK project\n\nCopyright (C) ANEO, 2022-$CURRENT_YEAR$. All rights reserved.\n  W. Kirschenmann   <wkirschenmann@aneo.fr>\n  J. Gurhem         <jgurhem@aneo.fr>\n  D. Dubuc          <ddubuc@aneo.fr>\n  L. Ziane Khodja   <lzianekhodja@aneo.fr>\n  F. Lemaitre       <flemaitre@aneo.fr>\n  S. Djebbar        <sdjebbar@aneo.fr>\n  J. Fonseca        <jfonseca@aneo.fr>\n  D. Brasseur       <dbrasseur@aneo.fr>\n  A. Louvion       <alouvion@aneo.fr>\n\nLicensed under the Apache License, Version 2.0 (the "License")\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an "AS IS" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License.\n
```

Comment (ddubuc) [reply .editorconfig:68]: add Arnaud in Aurthor list

**Review by ddubuc — CHANGES_REQUESTED** [review]

File: .editorconfig [hunk 2]
```diff
@@ -65,7 +65,7 @@ dotnet_style_qualification_for_field = false:suggestion
...
+file_header_template = This file is part of the ArmoniK project\n\nCopyright (C) ANEO, 2022-$CURRENT_YEAR$. All rights reserved.\n  W. Kirschenmann   <wkirschenmann@aneo.fr>\n  J. Gurhem         <jgurhem@aneo.fr>\n  D. Dubuc          <ddubuc@aneo.fr>\n  L. Ziane Khodja   <lzianekhodja@aneo.fr>\n  F. Lemaitre       <flemaitre@aneo.fr>\n  S. Djebbar        <sdjebbar@aneo.fr>\n  J. Fonseca        <jfonseca@aneo.fr>\n  D. Brasseur       <dbrasseur@aneo.fr>\n  A. Louvion       <alouvion@aneo.fr>\n\nLicensed under the Apache License, Version 2.0 (the "License")\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an "AS IS" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License.\n
```
Comment (ddubuc) [inline .editorconfig:68]:
```suggestion
file_header_template = This file is part of the ArmoniK project\n\nCopyright (C) ANEO, 2022-$CURRENT_YEAR$. All rights reserved.\n\nLicensed under the Apache License, Version 2.0 (the "License")\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an "AS IS" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License.\n
```

**Comment by aneojgurhem** [comment]
A PR that updates the files to incorporate these changes should also be done

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**